src/bench.ts

```diff
@@ -30,6 +30,19 @@ import {
   runtimeVersion,
 } from './utils'
 
+export interface Bench {
+  addEventListener<K extends BenchEvents>(
+    type: K,
+    listener: BenchEventsMap[K],
+    options?: AddEventListenerOptionsArgument
+  ): void;
+  removeEventListener<K extends BenchEvents>(
+    type: K,
+    listener: BenchEventsMap[K],
+    options?: RemoveEventListenerOptionsArgument
+  ): void;
+}
+
```

Comment:
I do not fully understand what you are trying to achieve with the EventTarget API redeclaration in two different places. 
It's type declaration duplication, the only difference is the event type and it could be consolidated. 
If it's to silence Deno, it's better to start from its type declaration for EventTarget and add it in a .d.ts to extend tinybench class type. Or to see if tsdown can include custom type generation depending on the runtime.